contracts/implementations/streamr/VehicleStream.sol

```diff
@@ -97,7 +98,8 @@ contract VehicleStream is AccessControlInternal {
 
         string memory streamId = string(
             abi.encodePacked(
```

Comment:
I'm curious, though—is there a difference between this and `string.concat`?

Reply:
The output is the same. But the `string.concat` is more expensive.

Reply:
Fascinating!